pages/golfer/[gid].js

```diff
@@ -0,0 +1,37 @@
+import { useRouter } from 'next/router'
+import Layout from '../../components/Layout'
+import ScoreCard from '../../components/ScoreCard'
+import useGolferScore from '../../lib/useGolferScore'
+
+const Profile = () => {
+  const router = useRouter()
+  const { gid } = router.query
+
+  const { golferScore, error } = useGolferScore(gid)
+
+  return (
+    <Layout>
+      <>
+        {error ? (
+          error
+        ) : (
+          <>
+            {golferScore?.name && <h1>{golferScore.name} scores:</h1>}
+            {golferScore?.scores && golferScore.scores.map(score => (
```

Comment:
When the data is not available it means it's loading, so if `golferScore` is `undefined` you can display a "Loading..." message.